css/main.css

```diff
@@ -1,3 +1,201 @@
 /* Add your custom CSS here */
+@import url('https://fonts.googleapis.com/css?family=PT+Sans&display=swap');
+body {
+    font-family: 'PT Sans', sans-serif;
+    background: rgb(219, 218, 218);
+    color: sienna;
+}
+/* custom gridfor about-us */
+.row-2 {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: 0px;
+    margin-left: 0px;
+  }
 
-/* Dont' forget to link this file to your HTML in the <head> */
+  .about-col-6 {
+    width: 100%;
+    position: relative;
+    min-height: 1px;
+    padding-right: 0px;
```

Comment:
if not definied, the padding (like margin) will be set to 0 - so this shouldn't be needed here